plugins/pa11y/src/tasks/pa11y.ts

```diff
@@ -0,0 +1,42 @@
+const pa11y = require('pa11y') // eslint-disable-line
+import { readState } from '@dotcom-tool-kit/state'
+import { Task } from '@dotcom-tool-kit/types'
+import { ToolKitError } from '@dotcom-tool-kit/error'
+import { Pa11ySchema } from '@dotcom-tool-kit/types/lib/schema/pa11y'
+
+export default class Pa11y extends Task<typeof Pa11ySchema> {
+  async run(): Promise<void> {
+    const reviewState = readState('review')
+    // if we've built a review app, test against that, not the app in the config
```

Comment:
**praise**: helpful comment! sorry I must have misunderstood it before - I thought it was bad practice to set this.options.host here, but now I can see why this.options.host is set here, it's so that this task can be run for review state by default, and if you've passed in host = 'https://ft-next-static-staging.herokuapp.com' for example you can run this task with staging